recipes/write_behind/oracle/rs/start-gear

```diff
@@ -1,6 +1,4 @@
 #!/bin/bash
 
-recipe=/opt/redislabs/lib/modules/gears_write_behind
-
-cd $recipe
+cd /opt/gears-wb
```

Comment:
Are you putting it there somewhere on the make part? We will once again will have permissions issue

Reply:
It's a Redis Enterprise path, relevant to a runtime environment, not a build one.